docs/platform-setup.md

```diff
@@ -18,3 +18,126 @@ For each job, disable the GitHub multibranch status:
 - Select "Disable GitHub Notifications".
 
 - Click on "Save".
+
+## Configure Nexus Docker Registry
+
+### Prerequisite
+
+- Nexus is installed.
+- NGINX Ingress Controller is installed.
+
+Configuring docker registry with nexus should follow this steps:
+
+### Create New Docker Repository
+
+Open the platform team's [Nexus](https://nexus.platform.dev.nuxeo.com/). Start new session using admin user
+
+#### Blob Storage And Repository Configuration
+
+- Create a blob storage : `docker-registry`
+
+  - In Server Administration and Configuration > Blob Stores,  click on the "Create blob store" button
+
+  - Complete all required fields and click on "Save"
+
+- Create a docker (hosted) nexus repository: `docker-registry`
+
+  - In Server Administration and Configuration > Repositories,  click on the "Create repository" button
+
+  - On storage section, use the previously created blob store
+
+  - Complete all required fields and click on "Save"
+
+### Cleanup Policy Configuration
+
+For each nexus docker registry, create a new policy cleanup and apply to it.
+
+#### Create New Cleanup Policy
+
+- In Server Administration and Configuration > Cleanup Policies,  click on the "Create Cleanup Policy" button
+
+- To keep all nuxeo builder and jenkins master images ( latest tag for example ) add this asset name matcher `~(.*nuxeo/builder-.*/manifests/latest|.*nuxeo/platform.*-jenkinsx/manifests/[0-9]{1,}.[0-9]{1,}.[0-9]{1,})`
+
+- Complete all required fields and click on "Save"
+
+#### Apply Cleanup Policy
+
+- Open an existing nexus repository configuration for instance [docker-registry](https://nexus.platform.dev.nuxeo.com/#admin/repository/repositories:docker-registry)
```

Comment:
I would say: "Open the [docker-registry](https://nexus.platform.dev.nuxeo.com/#admin/repository/repositories:docker-registry) repository configuration for instance."

Reply:
changes applied